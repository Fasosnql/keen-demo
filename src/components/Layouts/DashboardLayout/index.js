import React from 'react';

import Chart from 'components/Chart';
import {
  DataTextWrapper,
  DataChartWrapper,
  HeaderLine,
} from './styles';

const DashboardLayout = ({ pageViews }) => (
  <div>
    <DataTextWrapper>
      <HeaderLine>PAGE VIEWS:</HeaderLine>
      {pageViews.map((item) => (
        <div key={item.item}>
          {item.item}: {item.result}
        </div>
      ))}
    </DataTextWrapper>
    <DataChartWrapper>
      <Chart chartData={pageViews} />
    </DataChartWrapper>
  </div>
);

export default DashboardLayout;