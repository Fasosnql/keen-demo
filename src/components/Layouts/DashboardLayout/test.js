import React from 'react';
import DashboardLayout from './index';

import snapshotTest from 'utils/snapshot-test.util';

const pageViewsMock = [];
snapshotTest('Dashboard Layout renders correctly', (
  <DashboardLayout pageViews={pageViewsMock} />
));