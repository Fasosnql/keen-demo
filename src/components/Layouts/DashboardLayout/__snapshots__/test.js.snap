// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Layout renders correctly 1`] = `
<div>
  <div
    className="styles__DataTextWrapper-sc-9lj0p3-0 heakav"
  >
    <h5
      className="styles__HeaderLine-sc-9lj0p3-2 htGqst"
    >
      PAGE VIEWS:
    </h5>
  </div>
  <div
    className="styles__DataChartWrapper-sc-9lj0p3-1 dbgspN"
  >
    <div />
  </div>
</div>
`;
